ci/scripts/build-bitstream-vivado.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+usage() {
```

Comment:
This seems sufficiently complicated that perhaps a python script is in order. (Certainly, if you're parsing flags...)

Reply:
Well, we're not exactly parsing flags: there are just two strings that we're matching against a list. I *could* write a Python script, but I reckon that will be significantly more work. And you can't then run it with `bash -x ci/scripts/build-bitstream-vivado.sh`...